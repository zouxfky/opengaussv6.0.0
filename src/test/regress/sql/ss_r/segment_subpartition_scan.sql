SET CURRENT_SCHEMA TO segment_subpartition_scan;

select * from range_list order by 1, 2, 3, 4;

select * from range_list where month_code = '201902' order by 1, 2, 3, 4;
select * from range_list where dept_code = '1' order by 1, 2, 3, 4;
select * from range_list where user_no = '1' order by 1, 2, 3, 4;
