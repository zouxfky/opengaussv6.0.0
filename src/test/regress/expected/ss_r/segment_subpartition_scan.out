SET CURRENT_SCHEMA TO segment_subpartition_scan;
select * from range_list order by 1, 2, 3, 4;
 month_code | dept_code | user_no | sales_amt 
------------+-----------+---------+-----------
 201902     | 1         | 1       |         1
 201902     | 1         | 1       |         1
 201902     | 2         | 1       |         1
 201903     | 1         | 1       |         1
 201903     | 2         | 1       |         1
 201903     | 2         | 1       |         1
(6 rows)

select * from range_list where month_code = '201902' order by 1, 2, 3, 4;
 month_code | dept_code | user_no | sales_amt 
------------+-----------+---------+-----------
 201902     | 1         | 1       |         1
 201902     | 1         | 1       |         1
 201902     | 2         | 1       |         1
(3 rows)

select * from range_list where dept_code = '1' order by 1, 2, 3, 4;
 month_code | dept_code | user_no | sales_amt 
------------+-----------+---------+-----------
 201902     | 1         | 1       |         1
 201902     | 1         | 1       |         1
 201903     | 1         | 1       |         1
(3 rows)

select * from range_list where user_no = '1' order by 1, 2, 3, 4;
 month_code | dept_code | user_no | sales_amt 
------------+-----------+---------+-----------
 201902     | 1         | 1       |         1
 201902     | 1         | 1       |         1
 201902     | 2         | 1       |         1
 201903     | 1         | 1       |         1
 201903     | 2         | 1       |         1
 201903     | 2         | 1       |         1
(6 rows)

